stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
+                       (overlay-put overlay 'face stripe-highlight-face)
+                       (overlay-put overlay 'is-stripe t)
+                       (push overlay sb/overlays)))
+                  ((unless (dired-subtree--get-ov)
+                     (overlay-put overlay 'face stripe-highlight-face)
+                     (overlay-put overlay 'is-stripe t)
+                     (push overlay sb/overlays)))
+                  (t
+                     (overlay-put overlay 'face
+                                  (intern (format
+                                           "stripe-dired-subtree-depth-%d-face"
+                                           (cl-loop for x in (number-sequence 1 6)
```

Comment:
No.
This patch has faces until depth 6.
`dired-subtree--get-depth` indicate Infinite depth.